storagezone_get.go

```diff
@@ -0,0 +1,29 @@
+package bunny
+
+import "context"
+
+// StorageZone represents the response of the the List and Get Storage Zone API endpoint.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/storagezonepublic_index2 https://docs.bunny.net/reference/storagezonepublic_index
+type StorageZone struct {
+	ID *int64 `json:"Id,omitempty"`
+
+	UserID             *string     `json:"UserId,omitempty"`
+	Name               *string     `json:"Name,omitempty"`
+	Password           *string     `json:"Password,omitempty"`
+	DateModified       *string     `json:"DateModified,omitempty"`
+	Deleted            *bool       `json:"Deleted,omitempty"`
+	StorageUsed        *int64      `json:"StorageUsed,omitempty"`
+	FilesStored        *int64      `json:"FilesStored,omitempty"`
+	Region             *string     `json:"Region,omitempty"`
+	ReplicationRegions []string    `json:"ReplicationRegions,omitempty"`
+	PullZones          []*PullZone `json:"PullZones,omitempty"`
```

Comment:
I can confirm that its the exact same object. In the process of doing the diff I noticed that there are some missing properties from the existing `PullZone` object (perhaps added since the object was created). I'll add those as well.

Reply:
thanks a lot

Could you please create a separate PR to add the missing fields to the Pull-Zone? :-)
It's a different topic which we can process separately. 
I would like to keep the PR from getting bigger and bigger, it becomes more effort to review and 
 the actual functionality that you need (storage zone api) is getting delayed because of unrelated changes.